src/monitor.cc

```diff
@@ -714,7 +839,7 @@ bool NodeMonitor::sendReport() {
 
         for (int i=0; i < GCUsageTracker::kNumGCTypes; ++i) {
             v8::GCType type = GCUsageTracker::indexTov8GCType(i);
-            const GCStat stat = tracker.GetGCUsage( type )->EndInterval();
+            const GCStat stat = tracker.EndInterval( type );
```

Comment:
Is this `GCUsageTracker::GCUsage::EndInterval()`? That function doesn't take any arguments.